src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTests.cs

```diff
@@ -59,6 +59,230 @@ public async Task MapAction_SingleStringParam_StringReturn()
         await VerifyAgainstBaselineUsingFile(compilation);
     }
 
+    [Fact]
+    public async Task MapAction_SingleDateOnlyParam_StringReturn()
```

Comment:
Nit: To make migrating tests easier, it's better to move these related scenarios into a single `Theory` test case with member data.